src/Data/HaskellModule/Parse.hs

```diff
@@ -1,42 +1,45 @@
-{-# LANGUAGE OverloadedStrings, PackageImports, RecordWildCards #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE PackageImports    #-}
+{-# LANGUAGE RecordWildCards   #-}
 -- | In which a Haskell module is deconstructed into extensions and imports.
 module Data.HaskellModule.Parse (readModule) where
 
-import "mtl" Control.Monad.Trans
-import qualified Data.ByteString.Char8 as B
-import Data.List
-import Language.Haskell.Exts
-import Text.Printf
+import           Control.Applicative            ((<$>))
+import           "mtl" Control.Monad.Trans
+import qualified Data.ByteString.Char8          as B
+import           Data.List
+import           Language.Haskell.Exts
+import           Text.Printf
 
-import Control.Monad.Trans.Uncertain
-import Data.HaskellModule.Base
-import Data.HaskellSource
-import Language.Haskell.Exts.Location
+import           Control.Monad.Trans.Uncertain
+import           Data.HaskellModule.Base
+import           Data.HaskellSource
+import           Language.Haskell.Exts.Location
 
 
 locatedExtensions :: [ModulePragma] -> Located [ExtensionName]
 locatedExtensions = fmap go . located
   where
     go :: [ModulePragma] -> [ExtensionName]
     go = concatMap extNames
-    
+
     extNames :: ModulePragma -> [ExtensionName]
     extNames (LanguagePragma _ exts) = map prettyPrint exts
-    extNames (OptionsPragma _ _ _) = []  -- TODO: accept "-XExtName"
+    extNames OptionsPragma{}         = []  -- TODO: accept "-XExtName"
```

Comment:
Nice trick! Updating `Language/Haskell/Exts/Location.hs` to use this style might fix #149